README.md

```diff
@@ -1,17 +1,52 @@
 # Jolie Documentation
 
-Docker image with Gitbook using the [xiroV/gitbook-plugin-highlight-jolie](https://github.com/xiroV/gitbook-plugin-highlight-jolie) plugin, to support Jolie syntax highlighting.
+Documentation for Jolie `v1.10.x`.
+
+Before making a pull request, test your changes by building the documentation
+locally. Use the Docker image provided in this branch to build a Gitbook
+accessible on [localhost:8080](http://localhost:8080). Edit the files in the
+directory `web` and wait for Gitbook to reload the page to see your edits (be
+patient, it may take up to a minute).
+
+## Build the Docker image
+
+Docker image with Gitbook using the
+[highlight-jolie](https://github.com/xiroV/gitbook-plugin-highlight-jolie)
+plugin, to support Jolie syntax highlighting.
+
+### Using `docker-compose`
 
 Build the image with
 
 ```
-docker build -t jolie-docs . 
+docker-compose build
 ```
 
 Then run it with
 
 ```
-docker run --rm -it -p 8080:8080 -d jolie-docs
+docker-compose up -d
```

Comment:
I installed docker-compose, but I was unable to get this workflow working. It says `Starting docs_jolie-docs_1 ... done` upon running, but I cannot access the docs from the browser.

Reply:
I think the problem is that the book takes a while to build. Just wait a few minutes and then try to access the docs. Also try to run `docker-compose up` without the option `-d` to get some feedback from the running image.

Reply:
Thanks, I must have been too quick.